tests/languages/cli/test.js

```diff
@@ -1,6 +1,6 @@
 const { exec, execSync } = require('child_process');
 
-execSync("node index client --endpoint 'https://stage.appwrite.io/v1' --projectId console --key=35y3h5h345 --selfSigned true", { stdio: 'inherit' });
+execSync("node index client --endpoint 'https://demo.appwrite.io/v1' --projectId console --key=35y3h5h345 --selfSigned true", { stdio: 'inherit' });
```

Comment:
We need to decide on which instance we want to use for test, also that hast to have latest features even before release. We might want to maintain a separate instance I'm not sure yet but we need a better strategy.

Reply:
It should be suffice to use any Appwrite instance since this only tests the `/v1/health/version` endpoint. Anyways, it's better to get this working than to have it broken.